### DISCUSSION TITLE
two accounts on the same computer

### DISCUSSION THREAD

**Discussion by AllMontilla** [community/community#57131] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Good day, greetings, I have a github account and my wife has one too. How to use both accounts on a single computer using Git. Thank you.

**Comment by danielalyoshin** [comment]
You can set your global username and global email in git via a command line using:
`$ git config --global user.name "Your Name"`
`$ git config --global user.email "YOUR_EMAIL"`

You can then unset them using:
`$ git config --global --unset user.name`
`$ git config --global --unset user.email`

If you would like to do those actions but only for a single repo and not the entire computer, simply omit the `--global` flag and _ensure you are performing the command in the repo directory_.

For more information, here is the GitHub documentation for reference on setting your [username](https://docs.github.com/en/get-started/getting-started-with-git/setting-your-username-in-git) and [email](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address).

Hope this helps!

**Comment by karimdevelops** [comment]
I will recommend you setting up a different user on the OS you are using. 
You can have two accounts on the same computer, however, it will be hard to organize things. 

EDIT: If you are comfortable with your current environment then just follow what @danielalyoshin recommended

**Comment by semmakara6666** [comment]
Group's ***@***.***
